src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemFilingContextTests.cs

```diff
@@ -50,6 +50,17 @@ public void WorkItemFilingContext_NullSarifLogRaisesArgumentNullException()
             workItemModel.Area.Should().BeNull();
         }
 
+        [Fact]
+        public void WorkItemFilingContext_PopulateDescription()
```

Comment:
>WorkItemFilingContext_PopulateDescription [](start = 20, length = 41)

The name of this test s/be 'WorkItemFilingContext_PopulatesDescription'. Per Larry, the test name communicates what is accomplishes.